.github/workflows/test_R.yml

```diff
@@ -0,0 +1,58 @@
+name: test-R-ubuntu
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+jobs:
+  test:
+    name: Test on ubuntu-latest
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install ubuntu dependencies
+      run: sudo apt-get install -y libcurl4-openssl-dev qpdf
+
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install Python 3.x dependencies
+      run: |
+        python3 -m pip install --upgrade pip
+        python3 -m pip install numpy
+
+    - name: Install Metaflow python library
+      run: python3 -m pip install . 
+
+    - name: Set up R 
+      uses: r-lib/actions/setup-r@v1
+      with:
+        r-version: '3.6.3'
+
+    - name: Install Metaflow R binding and its dependencies
+      run: |
+        Rscript -e 'install.packages(c("R6", "reticulate", "magrittr", "lubridate", "cli"), repos="https://cloud.r-project.org")'        
+        cd R/; R CMD INSTALL .
```

Comment:
I tried to avoid using methods from `devtools` library here since `devtool` has a big dependency chain and very slow to install. But yeah it's much better to use `install_deps(.)`. Switching to `devtools::insall_deps('.')`